templates/dalite_settings.py

```diff
@@ -0,0 +1,92 @@
+import getpass
+
+import os
+
+SECRET_KEY = '{{ DALITE_SECRET_KEY }}'
+
+DEBUG = False
+
+ALLOWED_HOSTS = ['*']
```

Comment:
Why '*' instead of `DALITE_SERVER_DOMAIN` ? Isn't the latter better for security?